src/i18n.js

```diff
@@ -31,6 +31,7 @@ import contentTr from './locales/tr/content.json';
 import contentSm from './locales/sm/content.json';
 import contentAs from './locales/as/content.json';
 import contentUr from './locales/ur/content.json';
+import contentNp from './locales/np/content.json';
```

Comment:
the content.json file is empty.

Reply:
Sorry, That File Was Unsaved, I'm Uploading It Soon.

Reply:
I Have Added The content.json

Reply:
Nice. Great Job.